<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="mainWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7" rowstretch="1,1,20">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="frame_top">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5" columnstretch="13,6,10,2,1">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButton_login">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>838</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_station">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string> </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_CurUser">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_HomePage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,1">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="6,1">
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <property name="margin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTabWidget" name="tabWidget">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab">
               <attribute name="title">
                <string>智能告警</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_8">
                <property name="margin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QTableWidget" name="tableWidget_SmartAlarm">
                  <column>
                   <property name="text">
                    <string>告警时间</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>告警内容</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>原始告警</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_9">
                <property name="margin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QTableWidget" name="tableWidget_OriginalAlarm">
                  <column>
                   <property name="text">
                    <string>告警时间</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>告警内容</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="frame_scene">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="margin">
              <number>9</number>
             </property>
             <property name="spacing">
              <number>6</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_HistoryDetail">
         <layout class="QGridLayout" name="gridLayout_HistoryQuery">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
        <widget class="QWidget" name="page_AlarmPage">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QStackedWidget" name="stackedWidget_AlarmPage">
            <widget class="QWidget" name="page_RemoteControlProcess">
             <layout class="QGridLayout" name="gridLayout_RemoteControl">
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="page_SwitchStatusCheck">
             <layout class="QGridLayout" name="gridLayout_SwitchStatusCheck">
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="page_RemoteSignalling">
             <layout class="QGridLayout" name="gridLayout_RemoteSignalling">
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="page_RemoteMeasuring">
             <layout class="QGridLayout" name="gridLayout_RemoteMeasuring">
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="page_SingleBayProtectAction">
             <layout class="QGridLayout" name="gridLayout_SingleBayProtectAction">
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="page_SoeTimestampChange">
             <layout class="QGridLayout" name="gridLayout_SoeTimestampChange">
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="page_StationTimeSyncAbnormol">
             <layout class="QGridLayout" name="gridLayout_StationTimeSyncAbnormol">
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="page_DeviceCommInterrupt">
             <layout class="QGridLayout" name="gridLayout_DeviceCommInterrupt">
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="page_PrimaryEquipmentAbnormal">
             <layout class="QGridLayout" name="gridLayout_PrimaryEquipmentAbnormal">
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="page_SecondaryEquipmentAbnormal">
             <layout class="QGridLayout" name="gridLayout_SecondaryEquipmentAbnormal">
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="page_SwitchActionTimeCheck">
             <layout class="QGridLayout" name="gridLayout_SwitchActionTimeCheck">
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="page_PowerSystemAbnormal">
             <layout class="QGridLayout" name="gridLayout_PowerSystemAbnormal">
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="page_ElectricRunCheck">
             <layout class="QGridLayout" name="gridLayout_ElectricRunCheck">
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="page_AuxiliaryPowerCheck">
             <layout class="QGridLayout" name="gridLayout_AuxiliaryPowerCheck">
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_ConfigurationInterface">
         <layout class="QGridLayout" name="gridLayout_ConfigurationInterface">
          <property name="margin">
           <number>0</number>
          </property>
         </layout>
        </widget>
        <widget class="QWidget" name="page_StatisticAnalysis">
         <layout class="QGridLayout" name="gridLayout_StatisticAnalysis">
          <property name="margin">
           <number>0</number>
          </property>
         </layout>
        </widget>
        <widget class="QWidget" name="page_SystemConfig">
         <layout class="QGridLayout" name="gridLayout_SystemConfig">
          <property name="margin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="SwitchWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0,0,0,0,10">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_HomePage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>首页</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_ConfigurationInterface">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>画面显示</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_RealTime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="text">
            <string>实时告警</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButton_SystemConfig">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>系统管理</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButton_HistoryQuery">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>历史查询</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="pushButton_StatisticAnalysis">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>统计分析</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="pushButton_Help">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>帮助</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
